### PR TITLE
Appending multiple variants

### PR DESCRIPTION
**Pull request by pramodShehan5**

### CODE REVIEW

**Comment by pramodShehan5** [comment]
@adamw There is no way to prepend multiple errorVariants using errorOutVariantPrepend.
So I just created new function for it

File: core/src/main/scala/sttp/tapir/Endpoint.scala [hunk 1]
```diff
@@ -220,6 +220,10 @@ trait EndpointErrorOutputVariantsOps[A, I, E, O, -R] {
   def errorOutVariantPrepend[E2 >: E](o: OneOfVariant[_ <: E2]): EndpointType[A, I, E2, O, R] =
     withErrorOutputVariant(oneOf[E2](o, oneOfDefaultVariant(errorOutput)), identity)
 
+   /** Same as [[errorOutVariantPrepend]], but allows appending multiple variants in one go. */
+  def errorOutVariantPrepends[E2 >: E](first: OneOfVariant[_ <: E2], other: OneOfVariant[_ <: E2]*): EndpointType[A, I, E2, O, R] =
```
Comment (adamw) [inline core/src/main/scala/sttp/tapir/Endpoint.scala:224]: I think `errorOutVariantsPrepend` would be more grammatically correct?

Comment (pramodShehan5) [reply core/src/main/scala/sttp/tapir/Endpoint.scala]: I changed the function name :)

**Comment by adamw** [comment]
Thanks! :)

**Pull request merged**